Cargo.toml

```diff
@@ -1,4 +1,4 @@
-[workspace]
```

Comment:
Why removing this? Is it optional or something?

Reply:
I don't think this change is correct; it also breaks CI. Please revert.

Reply:
Yes, that was an incorrect change due to a previous hack that I missed in cleanup, reverted.